cloud/ovh/ovh_ip_loadbalancing_backend.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import sys
+try:
+    import ovh
+    import ovh.exceptions
+    HAS_OVH = True
+except ImportError:
+    HAS_OVH = False
+
+# import module snippets
+from ansible.module_utils.basic import *
+# and APIError from ovh api
+from ovh.exceptions import APIError
+
+DOCUMENTATION = '''
+---
+module: ovh_ip_loadbalancing_backend
+short_description: Manage OVH IP LoadBalancing backends
+description:
+    - Manage OVH (French European hosting provider) LoadBalancing IP backends
+version_added: "2.0"
+author: Pascal HERAUD @pascalheraud
+notes:
+    - Uses the python OVH Api U(https://github.com/ovh/python-ovh). \
+    You have to create an application (a key and secret) with a consummer \
+    key as described into U(https://eu.api.ovh.com/g934.first_step_with_api)
+requirements:
+    - ovh >  0.3.5
```

Comment:
it looks like ovh needs python >= 2.6.  you need to add that to the requirements section here.  (This is especially important because otherwise you'd need to code the module to be compatible with python-2.4 and I wouldn't wish that on you unnecessarily ;-)